### DISCUSSION TITLE
How to convert png from 24bits to 32bits

### DISCUSSION THREAD

**Discussion by veso266** [nothings/stb#1264] [Q&A]
Hello there, I was wondering on how to convert a 24bitmap png to 32bitmap png
using FreeImage you can use FreeImage_ConvertTo32Bits, but I didn't find any equivalent function in stb_image
if you have any idea how to do this using stb that would be great

Thanks for Anwsering and Best Regards

**Comment by nothings** [comment]
stb_image doesn't provide any way to do this separately, but you can do it while loading. pass in 4 for "desired_channels" and the load function will give you a 32-bit image
